components/experimental/src/dimension/provider/currency.rs

```diff
@@ -111,20 +113,20 @@ pub enum PlaceholderValue {
 )]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 #[derive(Copy, Debug, Clone, Default, PartialEq, PartialOrd, Eq, Ord)]
-pub struct CurrencyPatterns {
+pub struct CurrencyPatternConfig {
     /// If it is true, then use the standard pattern.
     /// Otherwise, use the standard_alpha_next_to_number pattern.
-    pub short_pattern_standard: PatternSelection,
+    pub short_pattern_selection: PatternSelection,
 
     /// If it is true, then use the standard pattern.
     /// Otherwise, use the standard_alpha_next_to_number pattern.
-    pub narrow_pattern_standard: PatternSelection,
+    pub narrow_pattern_selection: PatternSelection,
 
     /// The index of the short pattern place holder in the place holders list.
     /// If the value is `None`, this means that the short pattern does not have a place holder.
-    pub short_place_holder_index: Option<PlaceholderValue>,
+    pub short_placeholder_index: Option<PlaceholderValue>,
```

Comment:
Nit: make this `short_placeholder_value` since the type is `PlaceholderValue`

Reply:
(comment should have been on #4697)

Reply:
no problem,  I will fix it here

Reply:
done